packages/google_sign_in/google_sign_in_ios/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.3.0
+
+* Remove the requirement for configuring the app by adding a `GoogleService-Info.plist` file to the
```

Comment:
nits:

I don't think it is necessary to mention the first item. Maybe just as simple as
```
Add support for `serverClientId` configuration option.
```

Reply:
Done